eng/pipelines/coreclr/templates/helix-queues-setup.yml

```diff
@@ -34,7 +34,7 @@ jobs:
 
     # iOS Simulator/Mac Catalyst arm64
     - ${{ if in(parameters.platform, 'maccatalyst_arm64', 'iossimulator_arm64') }}:
-      - OSX.14.Arm64.Open
+      - OSX.26.Arm64.Open
```

Comment:
Consider using ` $(helix_macos_arm64)` if you just want to be on the "default" MacOS ARM64 queue, or `$(helix_macos_arm64_latest)` if you explicitly want to be on the latest.

Reply:
Thanks! I had to go back to osx.15 due to an issue with xharness, I'll go to `helix_macos_arm64` when we fix the iOS.26 support in xharness.